lib/TimezoneGuesser/FindFromOutlookCities.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabre\VObject\TimezoneGuesser;
+
+use DateTimeZone;
+
+class FindFromOutlookCities implements TimezoneFinder
+{
+    /**
+     * Example: TZID:(UTC+01:00) Bruxelles\, København\, Madrid\, Paris
+     */
+    public function find(string $tzid, bool $failIfUncertain = false): ?DateTimeZone
+    {
+        $values = explode(' ', $tzid);
+        if (count($values) === 1) {
+            return null;
+        }
+
+        unset($values[0]);
+        $cities = implode('', $values);
```

Comment:
Would be safer to do a regex for the UTC and remove the first matching group:
`^\(UTC(\+|\-)\d{2}:\d{2}\)`

Reply:
Why would it be safer?

Reply:
The current code would accept the timezone "Bruxelles\, København\, Madrid\, Paris" and throw away Bruxelles

Reply:
You are right! Updated, thanks!